src/Handler/CurlMultiHandler.php

```diff
@@ -53,7 +53,7 @@ class CurlMultiHandler
     /**
      * @var array<mixed> An associative array of CURLMOPT_* options and corresponding values for curl_multi_setopt()
      */
-    private $options = [];
+    private $options;
```

Comment:
No need to initialize this as we are already writing to it inside the constructor, as I don't assume the code is intended to be run with `\ReflectionClass::newInstanceWithoutConstructor()`

Reply:
i think this was added on purpose. please check the git blame